Websites/webkit.org/wp-content/themes/webkit/style.css

```diff
@@ -2297,6 +2298,10 @@ header .menu-item { /* add bottom dimension to main menu items */
     transition: 200ms ease-out width, 200ms ease-in background-color;
 }
 
+#site-nav .search-input {
```

Comment:
`#site-nav` only appears to be used in one other spot.  is this the best selector to use?

Reply:
Yes, it addresses a singularly specific case, exactly where an id-selector is appropriate. We're not defining a class of behaviors, but a very specific and scoped behavior.